internal/dependencytrack/project.go

```diff
@@ -31,9 +32,35 @@ type ProjectMetrics struct {
 	InheritedRiskScore float64 `json:"inheritedRiskScore"`
 }
 
-// GetProjects returns a list of all projects
-func (c *Client) GetProjects() ([]*Project, error) {
-	req, err := c.newRequest(http.MethodGet, "/api/v1/project", nil)
+// GetAllProjects returns a list of all projects with the help of pagination
+func (c *Client) GetAllProjects() ([]*Project, error) {
+	// dependency track per default only returns a 100 items per get, therefore we need to iterate over allProjects pages to get allProjects projects
+
+	// allProjects found in pagination
+	allProjects := []*Project{}
+	// the last project found in the last pagination page result
+	lastPaginationPage := 1
+	// state var to show if allProjects projects where found
+	foundAll := false
```

Comment:
```suggestion
// GetAllProjects returns a list of all projects
func (c *Client) GetAllProjects() ([]*Project, error) {
	allProjects := []*Project{}
	lastPaginationPage := 1
	foundAll := false
```

nit: I don't think the comments are necessarily. It's clear enough from the code.